src/components/DepositWithdraw/DepositWithdraw.tsx

```diff
@@ -103,6 +99,7 @@ export default function DepositWithdraw(props: IProps) {
   const [availableToWithdraw, setAvailableToWithdraw] = useState(BigNumber.from(0));
   const [withdrawAmount, setWithdrawAmount] = useState(BigNumber.from(0));
   const [userShares, setUserShares] = useState(BigNumber.from(0));
+  const apy = hatsPrice ? calculateApy(props.data.parentVault, hatsPrice, tokenPrice) : 0;
 
   useEffect(() => {
```

Comment:
i am not sure about the need for this data, @roienatan please let me know why this data is used and why there is polling here

Reply:
apy is the incentive mechanism. it should be present as we kick off the lm incentives